gosnmp.go

```diff
@@ -83,6 +83,22 @@ type GoSNMP struct {
 	// are ignored via shortcircuit
 	loggingEnabled bool
 
+	// Time keeping function handles allows one to pass in a function to use for timing.
+	// PreSend is called before a packet is sent
```

Comment:
Minor wording suggestion.

```suggestion
	// Message hook methods allow passing in a functions at various points in the packet handling.
	// For example, this can be used to collect packet timing, add metrics, or implement tracing.
	// PreSend is called before a packet is sent.
```